src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/Generated/ContainerServiceOperations.cs

```diff
@@ -1,873 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@brendandburns I'm assuming that this file was deleted for `ContainerServicesOperations.cs`, but that file does not exist